_component_design/toggle.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Toggle
```

Comment:
the thumbnail needs to change to more accurately represent a toggle. currently it appears just like a switch...

Reply:
That was my thinking too. Are you able to do that for me?

Reply:
Done.